doc/nrf/getting_started/installing.rst

```diff
@@ -389,16 +390,15 @@ Use the following commands to install the requirements for each repository.
 
 .. rst-class:: numbered-step
 
-Install a Toolchain
-*******************
-
-A toolchain provides a compiler, assembler, linker, and other programs required to build Zephyr applications.
+Install the Zephyr SDK
+**********************
 
 The Zephyr Software Development Kit (SDK) contains toolchains for each of Zephyr's supported architectures.
-It also includes additional host tools, such as custom QEMU and OpenOCD builds.
+Each toolchain provides a compiler, assembler, linker, and other programs required to build Zephyr applications.
```

Comment:
I would be very careful of how you use the word "toolchain"  in this section, because of the (IMHO not very adequate) use of "toolchain" by NCS. I would perhaps even add an item to the glossary and link to it!

toolchain: compiler, assembler and linker
Zephyr SDK: toolchain + some extra tools
NCS toolchain: Zephyr SDK  + all other programs required to build apps

Reply:
Also, this part here:
> and other programs required to build Zephyr applications.

is not quite true. Maybe "and some, but not all, of the rest of the programs required to build Zephyr applications"

Reply:
I made edits: added links to the glossary term for toolchain, which now describes the concept, the Zephyr SDK, and the NCS toolchain.